op_builder/builder.py

```diff
@@ -433,7 +433,7 @@ def builder(self):
 
     def load(self, verbose=True):
         from deepspeed.git_version_info import installed_ops, torch_info
-        if installed_ops[self.name]:
+        if self.name in installed_ops.keys() and installed_ops[self.name]:
```

Comment:
Could simplify as `if installed_ops.get(self.name, False)`

Reply:
Thanks for the suggestion, will change.